docs/3.api/2.composables/use-request-header.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "useRequestHeader"
+description: "Use useRequestHeader to access a certain incoming request header."
+links:
+  - label: Source
+    icon: i-simple-icons-github
+    to: https://github.com/nuxt/nuxt/blob/main/packages/nuxt/src/app/composables/ssr.ts
+    size: xs
+---
+
+You can use the built-in [`useRequestHeader`](/docs/api/composables/use-request-header) composable to access any incoming request header within your pages, components, and plugins.
+
+```ts
+// Get the cookie request header
+const cookie = useRequestHeader('cookie')
```

Comment:
I'd recommend using a different header as an example as we already have `useCookie`. Maybe `authorization`?

Reply:
Done, I added authorization in stead, thanks!